homeworks/hw1/Daniil_Sidorov/main.py

```diff
@@ -0,0 +1,145 @@
+import argparse
+
+import pandas as pd
+import torch
+import torch.nn as nn
+from sklearn.metrics import accuracy_score, confusion_matrix
+from sklearn.preprocessing import StandardScaler
+from sklearn.metrics import classification_report
+import matplotlib.pyplot as plt
+
+train_csv = pd.read_csv('../data/train.csv')
+val_csv = pd.read_csv('../data/val.csv')
+test_csv = pd.read_csv('../data/test.csv')
```

Comment:
also name dataframes as 'test_df' instead of 'test_csv', cause name 'test_csv' is usually used to name string values only (e.g. ''../data/test.csv'')